python/ray/tests/test_max_worker_default.py

```diff
@@ -0,0 +1,58 @@
+import pytest
```

Comment:
can you please move this test to test_autoscaler_yaml.py and remove this file and the entry from BUILD?